Connecting to database specified by database.yml
Connecting to database specified by database.yml


Started GET "/assets/rails.png" for 127.0.0.1 at 2013-10-30 17:59:22 -0300

PG::ConnectionBad (FATAL:  banco de dados "lojanet_production" não existe
):
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `connect'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:329:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `postgresql_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:309:in `new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:319:in `checkout_new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:241:in `block (2 levels) in checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `loop'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `block in checkout'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:233:in `checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:96:in `block in connection'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:95:in `connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:404:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:168:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:142:in `connection'
  activerecord (3.2.5) lib/active_record/query_cache.rb:67:in `rescue in call'
  activerecord (3.2.5) lib/active_record/query_cache.rb:61:in `call'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:473:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `_run__1756411027__call__1657065330__callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.5) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.5) lib/rails/rack/logger.rb:26:in `call_app'
  railties (3.2.5) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.1) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.1) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.5) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.1) lib/rack/lock.rb:15:in `call'
  rack-cache (1.2) lib/rack/cache/context.rb:136:in `forward'
  rack-cache (1.2) lib/rack/cache/context.rb:245:in `fetch'
  rack-cache (1.2) lib/rack/cache/context.rb:185:in `lookup'
  rack-cache (1.2) lib/rack/cache/context.rb:66:in `call!'
  rack-cache (1.2) lib/rack/cache/context.rb:51:in `call'
  railties (3.2.5) lib/rails/engine.rb:479:in `call'
  railties (3.2.5) lib/rails/application.rb:220:in `call'
  rack (1.4.1) lib/rack/content_length.rb:14:in `call'
  railties (3.2.5) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.1) lib/rack/handler/webrick.rb:59:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/" for 127.0.0.1 at 2013-10-30 17:59:40 -0300

PG::ConnectionBad (FATAL:  banco de dados "lojanet_production" não existe
):
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `connect'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:329:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `postgresql_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:309:in `new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:319:in `checkout_new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:241:in `block (2 levels) in checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `loop'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `block in checkout'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:233:in `checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:96:in `block in connection'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:95:in `connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:404:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:168:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:142:in `connection'
  activerecord (3.2.5) lib/active_record/query_cache.rb:67:in `rescue in call'
  activerecord (3.2.5) lib/active_record/query_cache.rb:61:in `call'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:473:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `_run__1756411027__call__1657065330__callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.5) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.5) lib/rails/rack/logger.rb:26:in `call_app'
  railties (3.2.5) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.1) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.1) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.5) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.1) lib/rack/lock.rb:15:in `call'
  rack-cache (1.2) lib/rack/cache/context.rb:136:in `forward'
  rack-cache (1.2) lib/rack/cache/context.rb:245:in `fetch'
  rack-cache (1.2) lib/rack/cache/context.rb:185:in `lookup'
  rack-cache (1.2) lib/rack/cache/context.rb:66:in `call!'
  rack-cache (1.2) lib/rack/cache/context.rb:51:in `call'
  railties (3.2.5) lib/rails/engine.rb:479:in `call'
  railties (3.2.5) lib/rails/application.rb:220:in `call'
  rack (1.4.1) lib/rack/content_length.rb:14:in `call'
  railties (3.2.5) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.1) lib/rack/handler/webrick.rb:59:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/" for 127.0.0.1 at 2013-10-30 17:59:41 -0300

PG::ConnectionBad (FATAL:  banco de dados "lojanet_production" não existe
):
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `connect'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:329:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `postgresql_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:309:in `new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:319:in `checkout_new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:241:in `block (2 levels) in checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `loop'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `block in checkout'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:233:in `checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:96:in `block in connection'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:95:in `connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:404:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:168:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:142:in `connection'
  activerecord (3.2.5) lib/active_record/query_cache.rb:67:in `rescue in call'
  activerecord (3.2.5) lib/active_record/query_cache.rb:61:in `call'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:473:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `_run__1756411027__call__1657065330__callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.5) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.5) lib/rails/rack/logger.rb:26:in `call_app'
  railties (3.2.5) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.1) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.1) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.5) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.1) lib/rack/lock.rb:15:in `call'
  rack-cache (1.2) lib/rack/cache/context.rb:136:in `forward'
  rack-cache (1.2) lib/rack/cache/context.rb:245:in `fetch'
  rack-cache (1.2) lib/rack/cache/context.rb:185:in `lookup'
  rack-cache (1.2) lib/rack/cache/context.rb:66:in `call!'
  rack-cache (1.2) lib/rack/cache/context.rb:51:in `call'
  railties (3.2.5) lib/rails/engine.rb:479:in `call'
  railties (3.2.5) lib/rails/application.rb:220:in `call'
  rack (1.4.1) lib/rack/content_length.rb:14:in `call'
  railties (3.2.5) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.1) lib/rack/handler/webrick.rb:59:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/home" for 127.0.0.1 at 2013-10-30 18:00:15 -0300

PG::ConnectionBad (FATAL:  banco de dados "lojanet_production" não existe
):
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `connect'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:329:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `postgresql_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:309:in `new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:319:in `checkout_new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:241:in `block (2 levels) in checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `loop'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `block in checkout'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:233:in `checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:96:in `block in connection'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:95:in `connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:404:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:168:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:142:in `connection'
  activerecord (3.2.5) lib/active_record/query_cache.rb:67:in `rescue in call'
  activerecord (3.2.5) lib/active_record/query_cache.rb:61:in `call'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:473:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `_run__1756411027__call__1657065330__callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.5) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.5) lib/rails/rack/logger.rb:26:in `call_app'
  railties (3.2.5) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.1) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.1) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.5) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.1) lib/rack/lock.rb:15:in `call'
  rack-cache (1.2) lib/rack/cache/context.rb:136:in `forward'
  rack-cache (1.2) lib/rack/cache/context.rb:245:in `fetch'
  rack-cache (1.2) lib/rack/cache/context.rb:185:in `lookup'
  rack-cache (1.2) lib/rack/cache/context.rb:66:in `call!'
  rack-cache (1.2) lib/rack/cache/context.rb:51:in `call'
  railties (3.2.5) lib/rails/engine.rb:479:in `call'
  railties (3.2.5) lib/rails/application.rb:220:in `call'
  rack (1.4.1) lib/rack/content_length.rb:14:in `call'
  railties (3.2.5) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.1) lib/rack/handler/webrick.rb:59:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'




Started GET "/home" for 127.0.0.1 at 2013-10-30 18:00:17 -0300

PG::ConnectionBad (FATAL:  banco de dados "lojanet_production" não existe
):
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:1206:in `connect'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:329:in `initialize'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `new'
  activerecord (3.2.5) lib/active_record/connection_adapters/postgresql_adapter.rb:28:in `postgresql_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:309:in `new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:319:in `checkout_new_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:241:in `block (2 levels) in checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `loop'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:236:in `block in checkout'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:233:in `checkout'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:96:in `block in connection'
  C:/Ruby200-x64/lib/ruby/2.0.0/monitor.rb:211:in `mon_synchronize'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:95:in `connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:404:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:168:in `retrieve_connection'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_specification.rb:142:in `connection'
  activerecord (3.2.5) lib/active_record/query_cache.rb:67:in `rescue in call'
  activerecord (3.2.5) lib/active_record/query_cache.rb:61:in `call'
  activerecord (3.2.5) lib/active_record/connection_adapters/abstract/connection_pool.rb:473:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:28:in `block in call'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `_run__1756411027__call__1657065330__callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:405:in `__run_callback'
  activesupport (3.2.5) lib/active_support/callbacks.rb:385:in `_run_call_callbacks'
  activesupport (3.2.5) lib/active_support/callbacks.rb:81:in `run_callbacks'
  actionpack (3.2.5) lib/action_dispatch/middleware/callbacks.rb:27:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/remote_ip.rb:31:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/debug_exceptions.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/show_exceptions.rb:56:in `call'
  railties (3.2.5) lib/rails/rack/logger.rb:26:in `call_app'
  railties (3.2.5) lib/rails/rack/logger.rb:16:in `call'
  actionpack (3.2.5) lib/action_dispatch/middleware/request_id.rb:22:in `call'
  rack (1.4.1) lib/rack/methodoverride.rb:21:in `call'
  rack (1.4.1) lib/rack/runtime.rb:17:in `call'
  activesupport (3.2.5) lib/active_support/cache/strategy/local_cache.rb:72:in `call'
  rack (1.4.1) lib/rack/lock.rb:15:in `call'
  rack-cache (1.2) lib/rack/cache/context.rb:136:in `forward'
  rack-cache (1.2) lib/rack/cache/context.rb:245:in `fetch'
  rack-cache (1.2) lib/rack/cache/context.rb:185:in `lookup'
  rack-cache (1.2) lib/rack/cache/context.rb:66:in `call!'
  rack-cache (1.2) lib/rack/cache/context.rb:51:in `call'
  railties (3.2.5) lib/rails/engine.rb:479:in `call'
  railties (3.2.5) lib/rails/application.rb:220:in `call'
  rack (1.4.1) lib/rack/content_length.rb:14:in `call'
  railties (3.2.5) lib/rails/rack/log_tailer.rb:17:in `call'
  rack (1.4.1) lib/rack/handler/webrick.rb:59:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:138:in `service'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/httpserver.rb:94:in `run'
  C:/Ruby200-x64/lib/ruby/2.0.0/webrick/server.rb:295:in `block in start_thread'


